pages/TBD/listings.html

```diff
@@ -26,7 +26,7 @@ <h1 class="text-center">Job Listings</h1>
               <input class="form-control" type="search" placeholder="Search Listings..." id="searchListing" data-search-listings />
               <span class="input-group-append">
                 <button class="btn ms-n5" type="button">
-                  <img src="/assets/icons/search.svg" alt="Search icon" class="icon-sm" />
+                  <img src="/public/assets/icons/search.svg" alt="Search icon" class="icon-sm" />
```

Comment:
Paths to images should not be changed untill the vite.config has been tried, goes for all path changes in this PR

Reply:
But that was the last ticket - not this one... I changed the link til search.svg in #454 Search Icons using wrong src

I can  understand if that one was not done correct, but how are that ticket related to this PR?

Reply:
Anyway - I understand someone else are working with this atm?

Reply:
The ticket #454 has been put on hold, this is correct. However the changes you did there, that we did not merge is also part of this pull request. It is in the commit history, and you can see it here under files changed. 

This PR is for #524 and should not contain the earlier fixes you did for #454